javascript/new-src/src/js/.jshintrc

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
not the same on has globals for nodejs (Gruntfile.js) and one for angular and jquery (app.js) so they are different